charon/tests/ui/issue-91-enum-to-discriminant-cast.out

```diff
@@ -0,0 +1,58 @@
+error: Unsupported statement kind: intrinsic
+  --> tests/ui/issue-91-enum-to-discriminant-cast.rs:16:13
+   |
+16 |     let _ = x as isize;
+   |             ^^^^^^^^^^
+
+[ INFO charon_driver::driver:338] [translate]: # Final LLBC before serialization:
+
+enum test_crate::Foo =
+|  A()
+|  B()
+
+
+trait core::clone::Clone<Self>
+{
+    fn clone : core::clone::Clone::clone
+    fn clone_from
+}
+
+trait core::marker::Copy<Self>
+{
+    parent_clause_0 : [@TraitClause0]: core::clone::Clone<Self>
+}
+
+fn test_crate::{impl core::clone::Clone for test_crate::Foo#1}::clone<'_0>(@1: &'_0 (test_crate::Foo)) -> test_crate::Foo
+{
+    let @0: test_crate::Foo; // return
+    let self@1: &'_ (test_crate::Foo); // arg #1
+
+    @0 := copy (*(self@1))
+    return
+}
+
+impl test_crate::{impl core::clone::Clone for test_crate::Foo#1} : core::clone::Clone<test_crate::Foo>
+{
+    fn clone = test_crate::{impl core::clone::Clone for test_crate::Foo#1}::clone
+}
+
+impl test_crate::{impl core::marker::Copy for test_crate::Foo} : core::marker::Copy<test_crate::Foo>
+{
+    parent_clause0 = test_crate::{impl core::clone::Clone for test_crate::Foo#1}
+}
+
+enum test_crate::Ordering =
+|  Less()
+|  Equal()
+|  Greater()
+
+
+fn test_crate::main()
+
+fn core::clone::Clone::clone<'_0, Self>(@1: &'_0 (Self)) -> Self
+
+
+
+error: aborting due to previous error
+
+[ ERROR charon_driver:264] The extraction encountered 1 errors
```

Comment:
Why does this one fail? I thought this PR would fix it.

Reply:
`as` casts use a different MIR statement, and hax currently doesn't support it